drivers/gpu/drm/lima/lima.h

```diff
@@ -149,6 +149,10 @@ int lima_gem_submit(struct drm_file *file, struct lima_sched_pipe *pipe,
 		    struct drm_lima_gem_submit_bo *bos, u32 nr_bos,
 		    void *frame, u32 *fence);
 int lima_gem_wait(struct drm_file *file, u32 handle, u32 op, u64 timeout_ns);
+struct drm_gem_object *
+lima_gem_prime_import_sg_table(struct drm_device *dev,
```

Comment:
Sorry for my miss leading comments, I really mean new line arg align with the start of the last line arg not the "(", so in this case, "struct" align with the "struct". Please refer to my other functions which have multi line of arg.